docs/v2-release-notes.md

```diff
@@ -2,6 +2,17 @@
 
 This page summarizes changes for Timeplus Enterprise and Timeplus Proton, on a biweekly basis, including new features and important bug fixes.
 
+## Aug 6, 2024
+ * Timeplus Enterprise v2.4.15 is released. [See full changelog](enterprise-releases)
+ * Timeplus Proton v1.5.15 is released, allowing Timeplus Enterprise v2.4 to read or write via external streams. [Learn more](timeplus-external-stream)
+ * [timeplus-natve-jdbc](jdbc) v2.0.3 and v2.0.4 now available.
+    * Bug fix: For low_cardiality(nullable), nullable(uuid), map(low_cardinality) and tuple(low_cardinality)
+    * Bug fix: Fixed logging problem in some datatypes
+    * Bug fix: For the map defaultValue()
+ * Improved error handling in the SQL Console: if errors occur when executing the query, runtime errors and intermediate query results are shown.
```

Comment:
these 3 lines are related to Timeplus Enterprise. Should be put under 2.4.x

Reply:
Hi @jovezhong, updated and brought back the Enterprise/Proton categories